### DISCUSSION TITLE
Improve plugin discoverability by adding sort by downloads, popularity, or recommended

### DISCUSSION THREAD

**Discussion by LiamSmyth** [godotengine/godot-proposals#6329] [Editor]
Hello! Just brainstorming some ways to make GD assetlib more discoverable:

I've been looking around for some widely-useful plugins on the assetLib and am finding it difficult to find the most generically useful plug-ins because the only sorting options are name/license/recently updated. It would be great if there was a way to sort by popularity or download count, so that I could see which are the most production-ready or generally useful addons. Another thing that might help would be a way to keep one-shots or novelty addons out of my searches, as it is the windows-98 Godot theme bubbles as high in the list as visual behavior tree addon. No problem with the W98 theme, it just isn't what I'm looking for!

Alternatives could be hand-curated plugin bundles that can be submitted by community members, for doing particular tasks. EG the "RTS plugin bundle" -- just a list of addons that could be useful to someone trying to make an RTS. Other ideas, time horizon for popular plugins, eg 'popular this week', 'popular this month', etc to see new and up-coming developments in the addon space!

**Comment by Calinou** [comment]
The asset library doesn't track downloads, so it's not feasible to provide sorting by download count.

**Reply by LiamSmyth** [reply]
Is there some other way for the best addons to bubble to the top of the list?

**Reply by Calinou** [reply]
> Is there some other way for the best addons to bubble to the top of the list?

Unfortunately, not without making changes to both the asset library and the editor integration. We don't have active maintainers for the [asset library backend](http://github.com/godotengine/asset-library) due to lack of time.
